pkgs/by-name/de/degit-rs/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "degit-rs";
+  version = "0.1.2-unstable-2021-09-22";
+
+  src = fetchFromGitHub {
+    owner = "psnszsn";
+    repo = "degit-rs";
+    rev = "c7dbeb75131510a79400838e081b90665c654c80";
+    hash = "sha256-swyfKnYQ+I4elnDnJ0yPDUryiFXEVnrGt9xHWiEe6wo=";
+  };
+
+  sourceRoot = finalAttrs.src.name;
+
+  postPatch = ''
+    # The source repo doesn't provide a Cargo.lock file, so we need to create one
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
```

```suggestion
  # The source repo doesn't provide a Cargo.lock file, so we need to create one
  postPatch = ''
    ln -s ${./Cargo.lock} Cargo.lock
  '';
```




Try to avoid putting comments in nix string.
Every time we change them the package will need to be rebuilt.